hack/jenkins/e2e.sh

```diff
@@ -212,6 +212,7 @@ case ${JOB_NAME} in
           ${GCE_DEFAULT_SKIP_TESTS[@]:+${GCE_DEFAULT_SKIP_TESTS[@]}} \
           ${GCE_PARALLEL_SKIP_TESTS[@]:+${GCE_PARALLEL_SKIP_TESTS[@]}} \
           ${GCE_PARALLEL_FLAKY_TESTS[@]:+${GCE_PARALLEL_FLAKY_TESTS[@]}} \
+          ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
```

Comment:
maybe expand this inside the def'n of `GCE_PARALLEL_FLAKY_TESTS` instead? we already do that for `GCE_PARALLEL_SKIP_TESTS` with `GCE_DEFAULT_SKIP_TESTS` (which probably means we don't need to have the latter here either)